src/views/reset-password/request-password-reset.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Card, Help, InputField, LogoIcon } from '@dhis2/ui'
+import { useAlert } from '@dhis2/app-service-alerts';
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useAuthAxios } from '../../hooks'
+import styles from './reset-password.module.css'
+
+export const RequestPasswordReset = () => {
+    const navigate = useNavigate()
+    const [inputs, setInputs] = useState({
+        email: '',
+    })
+    const [errorMessage, setErrorMessage] = useState('')
+    const { show: showAlert } = useAlert(
+        ({ message }) => message,
+        ({ isCritical }) => (isCritical ? { critical: true } : { success: true })
+    )
+    const onInputChange = useCallback(({ value, name }) => {
+        // Always clear the error when user starts typing again
+        setErrorMessage('')
+        setInputs((currentInputs) => ({
+            ...currentInputs,
+            [name]: value,
+        }))
+    }, [])
+    const [{ loading }, postRequestReset] = useAuthAxios(
+        {
+            url: '/users/request-reset',
```

Comment:
I'm pretty sure we're not doing this everywhere but I made it a (pedantic) habit to write the method before the url since that's how it's done in the protocol